integration/test/config-test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+/* eslint max-len: "off" */
+/* eslint no-return-assign: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const {version} = require('../package.json');
+
+const TestUtil = require('../test-util');
+const util = new TestUtil();
+
+describe('Configuration', function() {
+  this.timeout(5000);
+
+  before(async () => {
+    await util.open();
+  });
+
+  after(async () => {
+    await util.close();
+  });
+
+  describe.only('--agent', function () {
```

Comment:
`.only` to be removed? All tests pass without this.

Reply:
yep thanks